openvdb_points_houdini/houdini/VRAY_OpenVDB_Points.cc

```diff
@@ -394,10 +518,52 @@ VRAY_OpenVDB_Points::render()
         gdp->addTuple(GA_STORE_REAL32, GA_ATTRIB_POINT, "pscale", 1, GA_Defaults(DEFAULT_PSCALE));
     }
 
+    // map speed to color if requested
+    if (mSpeedToColor) {
+        for (PointDataGridPtrVecIter    iter = mGridPtrs.begin(),
+                                        endIter = mGridPtrs.end(); iter != endIter; ++iter) {
+
+            PointDataGridPtr grid = *iter;
+
+            PointDataTree& tree = grid->tree();
+
+            PointDataTree::LeafIter leafIter = tree.beginLeaf();
+            if (!leafIter) continue;
+
+            if (leafIter->hasAttribute("Cd"))   dropAttribute(tree, "Cd");
```

Comment:
Ah I see why you're doing this now. Yes, I would suggest templating CreateColorFromVelocity on the Color type, then support Vec3h or Vec3f. If Cd is neither, then drop it and re-add (but explain why you're doing this in a comment).

Perhaps the name "CreateColorFromVelocity" is a little misleading as well, because the color attribute has already been created by the time this class gets used (it just hasn't been populated).